.github/workflows/test.yml

```diff
@@ -487,6 +487,42 @@ jobs:
         with:
           projectBaseDir: schema/
 
+  test-indexer-postgres:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.22"
```

Comment:
nit, "1.12"

Reply:
I can't run the test go mod with 1.12

Reply:
Why not? It works for `schema`

Reply:
Oh I see, it is go 1.22, makes sense. I hope it complains if we add something in `indexer/postgres` that doesn't support go 1.12 then. Maybe we should add another job only for indexer that runs on 1.12 just for that purpose?

Reply:
Maybe we can have one job for schema + postgres/indexer together on go 1.12 and another job for indexer tests on 1.22? Or three separate jobs if needed